reliability-engineering/terraform/modules/concourse-team-iam-credentials/cred-rotation.tf

```
@@ -0,0 +1,56 @@
+data "aws_iam_role" "lambda_execution" {
+  name = var.lambda_execution_role_name
+}
+
```

Can we not have an zip file in the repo and use the terraform archive file provider instead? ⬇️ 

```suggestion

data "template_file" "sts_lambda_function_py" {
  template = "${file("${path.module}/files/sts-lambda-function.py")}"
}

data "archive_file" "sts_lambda" {
  type        = "zip"
  output_path = "${path.module}/files/sts-lambda.zip"

  source {
    content  = "${data.template_file.sts_lambda_function_py.rendered}"
    filename = "lambda_function.py"
  }
}

```

We would need to extract the python code from the archive, and put it in the repository under the filename `sts-lambda-function.py`

---

yes this would work 👍 

I'm happy editing python directly inside the zip file (emacs makes this easy) but your approach is friendlier to a greater spectrum of tooling (including github)